wagtailautocomplete/views.py

```diff
@@ -1,3 +1,8 @@
+try:
+    from urllib.parse import unquote
+except ImportError:
+    from urlparse import urlparse as unquote
```

Comment:
I _think_ this will cause trouble in Python 2.7. `urlparse.urlparse` [seems to return a 6-tuple](https://docs.python.org/2/library/urlparse.html#urlparse.urlparse). I think you might want [`urllib.unquote`](https://docs.python.org/2/library/urllib.html#urllib.unquote)

```suggestion
    # Python 2.7 compat
    from urllib import unquote
```